src/components/loadflow-result.js

```diff
@@ -53,7 +53,14 @@ const LoadFlowResult = ({ result, studyUuid, nodeUuid }) => {
         Number(state[PARAM_LIMIT_REDUCTION])
     );
     const loadflowNotif = useSelector((state) => state.loadflowNotif);
-
+    const LIMIT_TYPES = useMemo(
```

Comment:
I think there is no need to call useMemo here since there are no dependencies

Reply:
I think the best solution is to create this const out of the component